### PR TITLE
Update test.yml

### PR DESCRIPTION
**Pull request by XhmikosR**
* specify `CI:true`
* move Node.js version to an environment variable
* remove info which is now printed by actions/setup-node

### CODE REVIEW

**Review by Trott — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+
 on:
   push:
     branches-ignore:
       - "translations"
       - "dependabot/**"
   pull_request:
 
+env:
+  CI: true
```
Comment (nschonni) [inline .github/workflows/test.yml]: Already set by default https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

**Review by nschonni — COMMENTED** [review]
👍 I like the removal of those `--version` steps.
Don't really mind the node version variable, but that data is really already in there too in the setup-node step as you mention in the discription
```
/opt/hostedtoolcache/node/12.18.4/x64/bin/node --version
v12.18.4
/opt/hostedtoolcache/node/12.18.4/x64/bin/npm --version
6.14.6
```
Is it work using https://github.com/actions/setup-java if we care about the particular Java version?

File: .github/workflows/test.yml (hunk 1 above)
Comment (XhmikosR) [reply .github/workflows/test.yml]: Oh, good, I guess they finally added it, because it wasn't the case in the beginning.

**Comment by XhmikosR** [comment]
The default Java version should be fine :)

**Pull request merged**